.eslintrc.js

```diff
@@ -3,6 +3,13 @@ module.exports = {
   rules: {
     'no-warning-comments': 'warn', // we have many Todo:, this will remind us to deal with them
     'no-use-before-define': 'off',
+    'vue/attribute-hyphenation': [
+      'error',
+      'always',
+      {
+        ignore: ['createURL'],
```

Comment:
I've ignore this one for simplicity and coherence between the connector & Vue component. Otherwise the prop name would be `createUrl` rather than `createURL`.